src/compiler/transformers/module/system.ts

```diff
@@ -339,37 +339,6 @@ namespace ts {
                 }
             }
 
-            for (const externalImport of moduleInfo.externalImports) {

```

Comment:
I didn't see any tests affected that use SystemJS, so I'm not sure what effect (if any) this change made. Is it handled by the change in utilities.ts, or are we missing test cases?

Reply:
This removed code duplicated the reexports preservation for systemjs. In effect, for systemjs only, we were tracking reexports in a second list. So, rather than duplicating the systemjs code into all the other module systems, I removed code from the systemjs resolver and let it rely on the single list having the reexports in it (since every module mode needs them now).